src/server/utils/css.ts

```diff
@@ -33,7 +31,7 @@ export default `
   }
 
   .tittel-og-personinfo {
-    margin-top: 200px;
+    margin-top: 80px;
```

Comment:
Nå når logo og brevdato faktisk tar plass på siden, måtte `margin-top` for brevtittel justeres ned for å ikke få for stort mellomrom mellom logo og brevtittel.